packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -0,0 +1,116 @@
+:local {
+  .control {
+    color: var(--terra-form-checkbox-font-color, rgb(0, 0, 0));
+    display: inline-block;
+    font-family: inherit;
+    margin-right: var(--terra-form-checkbox-margin-right, 2em);
+    margin-top: var(--terra-form-checkbox-margin-top, 0.625em);
+    vertical-align: middle;
+
+    .label-text {
+      font-size: var(--terra-form-checkbox-font-size, 1rem);
+    }
+
+    /* Hide actual input element */
+    input:not(:checked),
+    input:checked {
+      border: 0;
+      clip: rect(0 0 0 0);
+      height: 1px;
+      margin: -1px;
+      overflow: hidden;
+      padding: 0;
+      position: absolute;
+      width: 1px;
+    }
+    /* stylelint-disable selector-max-compound-selectors */
+    input:not(:checked) + .label-text,
+    input:checked + .label-text {
+      cursor: pointer;
+      display: flex;
+      padding-left: var(--terra-form-checkbox-padding-left, 1.6em);
+      position: relative;
+    }
+
+    /* Checkbox aspect */
+    input:not(:checked) + .label-text::before,
```

Comment:
All of your selectors seem to be overly complex. You should be able to just use `.label-test::before` here.